### ISSUE TITLE
[X] Accessibility: Logo alt text is just "logo", should be descriptive

### ISSUE THREAD

**Issue by mtedeschi** [InteractiveMechanics/qa-training-repo#3]
**Description**
The alt text on the logo is just "logo" and should be "West Philadelphia History Map". Reported by WAVE.

**Screenshot**
<img width="373" alt="screen shot 2017-06-15 at 4 09 33 pm" src="https://user-images.githubusercontent.com/842387/27199766-380414f8-51e5-11e7-8021-c6fafa29f182.png">
